lms/static/js/learner_dashboard/views/entitlement_unenrollment_view.js

```diff
@@ -113,12 +117,65 @@ class EntitlementUnenrollmentView extends Backbone.View {
     this.$submitButton.data('entitlementApiEndpoint', apiEndpoint);
   }
 
+  switchToSlideOne() {
+    // Randomize survey option order
+    const survey = document.querySelector('.options');
+    for (let i = survey.children.length - 1; i >= 0; i -= 1) {
+      survey.appendChild(survey.children[Math.trunc(Math.random() * i)]);
+    }
+    console.log('in switch to slide1')
+    console.log('grabbing inner header: ', this.$('.entitlement-unenrollment-modal-inner-wrapper header'))
+    this.$('.entitlement-unenrollment-modal-inner-wrapper header').hide();
+    this.$('.entitlement-unenrollment-modal-submit-wrapper').hide();
```

Comment:
Display should be controlled by adding/removing classes instead of the jQuery ```show```/```hide``` functions due to a performance issue they have.